api/src/main/java/io/opentelemetry/trace/Link.java

```diff
@@ -29,13 +29,14 @@
  */
 @ThreadSafe
 public interface Link {
-  /**
-   * Returns the {@code SpanContext}.
-   *
-   * @return the {@code SpanContext}.
-   * @since 0.1.0
-   */
-  SpanContext getContext();
+
+  String getTraceIdAsHexString();
+
+  String getSpanIdAsHexString();
+
+  TraceState getTraceState();
+
+  byte getTraceFlags();
```

Comment:
This actually seems worse than before. Could we return a DefaultSpan here instead?

Reply:
`Link` is strange in being over-coupled to export right now. I don't think export structures should be based on e.g., `Span` so I'll look at it in #1714.